config/locales/en.yml

```diff
@@ -1926,6 +1926,7 @@ en:
         -4011: "You can only accept the first person on the waiting list"
         -4012: "You are not qualified to compete in the selected events"
         -4014: "You are already registered for this competition."
+        -4015: "Please enter a reasonable amount of guests, or contact the organizers if you need to bring an extraordinary amount of guests."
```

Comment:
This message should make it clear that the competitor should register with 9 guests now, and also email the organizers. They might miss a spot if they don't register now, and even if not, the organizers can't change the site to allow them to register with >9 guests.

Reply:
See other comment